src/registrar/tests/common.py

```diff
@@ -813,6 +827,29 @@ def dummyInfoContactResultData(
         addrs=["1.2.3.4", "2.3.4.5"],
```

Comment:
Reminder that the format of addrs will need to be updated once issue 1016 is merged.